internal/apps/ai-proxy/filters/session-context/filter.go

```diff
@@ -37,6 +37,10 @@ import (
 
 const (
 	Name = "session-context"
+
+	systemMessage = "Your name is Erda Assistant. You are trained by Erda. " +
```

Comment:
Make it configurable.
And I found if you add session topic(such as: `test`) after the systemMessage, it's strange and the effect is not good enough.
Maybe you should split systemMessage into:
```
background: xxx
session topic: yyy
```